scipy/interpolate/_rgi.py

```diff
@@ -63,8 +64,13 @@ class RegularGridInterpolator:
         strictly ascending or descending.
 
     values : array_like, shape (m1, ..., mn, ...)
-        The data on the regular grid in n dimensions. Complex data can be
-        acceptable.
+        The data on the regular grid in n dimensions. Complex data is
+        accepted.
+
+        .. deprecated:: 1.13.0
+            Complex data is deprecated with method "pchip" and will raise an
+            error in SciPy 1.15.0. This is because ``PchipInterpolator`` only
+            works will real values.
```

Comment:
Typo + what users should do instead
```suggestion
            Complex data is deprecated with ``method="pchip"` and will raise an
            error in SciPy 1.15.0. This is because ``PchipInterpolator`` only
            works with real values. If you are trying to use the real components of
            the passed array, use `np.real` on `values`.
```